src/NavDropdown.tsx

```diff
@@ -77,28 +82,37 @@ const NavDropdown: NavDropdown = (React.forwardRef(
       ...props
     }: NavDropdownProps,
     ref,
-  ) => (
-    <Dropdown ref={ref} {...props} as={NavItem}>
-      <Dropdown.Toggle
-        id={id}
-        eventKey={null}
-        active={active}
-        disabled={disabled}
-        childBsPrefix={bsPrefix}
-        as={NavLink}
-      >
-        {title}
-      </Dropdown.Toggle>
-
-      <Dropdown.Menu
-        role={menuRole}
-        renderOnMount={renderMenuOnMount}
-        rootCloseEvent={rootCloseEvent}
+  ) => {
+    bsPrefix = useBootstrapPrefix(bsPrefix, 'nav-item');
+
+    return (
+      <Dropdown
+        ref={ref}
+        {...props}
+        as={props.as ? props.as : NavItem}
```

Comment:
You can remove the `as={...}` line.  Since you are adding in the `.nav-item` class already, we don't need to use `NavItem` anymore.

Reply:
You are right! Thanks, Kyle!